docs/src/pages/docs/frequently-asked-questions-page.mdx

```diff
@@ -232,6 +232,30 @@ Metadata attribute using the `label_colors` key.
 }
 ```
 
+### Is there a way to add custom color schemes?
+
+It is possible to override EXTRA_CATEGORICAL_COLOR_SCHEMES or EXTRA_SEQUENTIAL_COLOR_SCHEMES to register new color schemes
+
+```
+EXTRA_CATEGORICAL_COLOR_SCHEMES =  [
+    {
+        "id": 'myCustomColors',
+        "description": '',
+        "label": 'My custom Colors',
+        "colors": ['#6BD3B3', '#6BD3B3', '#6BD3B3', '#66CBE2']
+    }]
+
+EXTRA_SEQUENTIAL_COLOR_SCHEMES =  [
+    {
+        "id": 'myCustomColors',
+        "description": '',
+        "isDiverging": true
+        "label": 'My custom Colors',
+        "colors": ['#6BD3B3', '#6BD3B3', '#66CBE2']
+    }]
```

Comment:
I would rather see this example in `config.py` with a reference to it in the FAQ. Also a minor nit, but would be nice to have the schemes having real colors and not repeating `#6BD3B3`, also with the name something like "My custom warm to hot" or similar.